FireRender.Maya.Src/FireRenderIBL.cpp

```diff
@@ -241,6 +241,16 @@ void FireRenderIBL::postConstructor()
 	MObject mobj = thisMObject();
 	m_attributeChangedCallback = MNodeMessage::addAttributeChangedCallback(mobj, FireRenderIBL::onAttributeChanged, this, &status);
 	assert(status == MStatus::kSuccess);
+
+	// rename node
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRIBLShape");
```

Comment:
Is there could be only one IBL Shape with that name on the scene?

Reply:
There could be only one RPRIBL object on the scene